### DISCUSSION TITLE
Is it possible to style/add text to the title page in an epub book

### DISCUSSION THREAD

**Discussion by richardtc** [quarto-dev/quarto-cli#6881] [Q&A]
### Description

Hi,
I'd like to create a title page in an epub book that looks similar to a PDF title page output.

Is there a way to style the epub title page - such as adding additional text, inserting an image?

Thanks for your advice.

I'm using Quarto version 1.3.353

**Comment by mcanouil** [comment]
I never used that format but I don't think there is currently a way within Quarto.

Maybe take a look at <https://pandoc.org/MANUAL.html#epubs>.

**Comment by cderv** [comment]
You could try customizing the default EPUB3 template: 
https://github.com/jgm/pandoc/blob/main/data/templates/default.epub3

and passing it using `template` options (https://quarto.org/docs/reference/formats/epub.html#rendering)
